### PR TITLE
Call out the need for poster url encoding

### PR DESCRIPTION
**Pull request by third774**
cc @zaidf

### CODE REVIEW

**Review by bkrebsbach — APPROVED** [review]

**Review by Everlag — APPROVED** [review]

**Review by zaidf — APPROVED** [review]

File: products/stream/src/content/viewing-videos/using-the-stream-player.md [hunk 1]
```diff
@@ -107,6 +107,12 @@ Player options are configured with querystring parameters in the iframe's `src`
 
   - A URL for an image to be shown before the video is started or while the video is downloading. If this attribute isn't specified, a thumbnail image of the video is shown.
 
+    <Aside>
```
Comment (marciocloudflare) [inline products/stream/src/content/viewing-videos/using-the-stream-player.md]: add `type="note"` to aside;
consider adding `**Note**` as title of Aside to make it clear this is a note.

File: products/stream/src/content/viewing-videos/using-the-stream-player.md [hunk 2]
```diff
@@ -107,6 +107,12 @@ Player options are configured with querystring parameters in the iframe's `src`
...
+
+      Like all query string parameters, this value _must_ be URI encoded. For example, the thumbnail at `https://videodelivery.net/5d5bc37ffcf54c9b82e996823bffbb81/thumbnails/thumbnail.jpg?time=68s&height=270` can be encoded using JavaScript's `encodeURIComponent()` function to `https%3A%2F%2Fvideodelivery.net%2F5d5bc37ffcf54c9b82e996823bffbb81%2Fthumbnails%2Fthumbnail.jpg%3Ftime%3D68s%26height%3D270`.
```
Comment (marciocloudflare) [inline products/stream/src/content/viewing-videos/using-the-stream-player.md]: I'm not sure of the meaning of the sentence `can be encoded using JavaScript's "encodeURIComponent()" function to`. Is there something missing after the `to`?

**Comment by marciocloudflare** [comment]
Hey @third774, could you take a look at my comments? Thank you!

File: products/stream/src/content/viewing-videos/using-the-stream-player.md (hunk 2 above)
Comment (third774) [reply products/stream/src/content/viewing-videos/using-the-stream-player.md]: `to` is referring to the encoded URL. The intention of the sentence is to demonstrate an example of how to go from an un-encoded URL to an encoded one.

Comment (marciocloudflare) [reply products/stream/src/content/viewing-videos/using-the-stream-player.md]: Got it. Then this makes perfect sense. Thanks!

**Pull request merged**